setup.py

```diff
@@ -29,7 +29,7 @@
     maintainer="Terri Oda",
     maintainer_email="terri.oda@intel.com",
     url="https://github.com/intel/cve-bin-tool",
-    license="GPLv3",
+    license="GPL3.0 or later",
```

Comment:
```suggestion
    license="GPL-3.0-or-later",
```

Hm, looking at this again, I'm wondering if we might as well switch to using the SPDX short identifier while we're making this change so the results will be more machine readable.  Here's a link: https://spdx.org/licenses/GPL-3.0-or-later.html

Pinging @anthonyharrison for a second opinion.   I haven't delved too deeply into this end of things yet but I will likely be doing so over the coming weeks.